Cargo.toml

```diff
@@ -30,7 +30,7 @@ members = [
   "crates/sdk",
   "crates/cuda",
   "crates/stark",
-  "crates/zkvm/*",
+  "crates/zkvm/*", "tests/uma-test", 
```

Comment:
why is this not on a new line like the rest of the the crates?

Also, do you see any other tests being included here? why did you add it. try to understand how the other tests work

Reply:
Yeah I was a bit stuck on that. I think it added it automatically when I added the uma-test package to the tests directory. Not sure how to add it without it getting added to the root Cargo.toml in this way